test/hyperdeck/parser.js

```diff
@@ -1,25 +1,47 @@
 var Parser = require('../../src/hyperdeck/parser');
 
-var SUCCESS_RESPONSE = "200 Success";
-var SUCCESS_RESPONSE_WITH_PARAMS = "200 Success with data:\r\nsomething: 123\r\nsomething else: test\r\n\r\n";
-var SUCCESS_RESPONSE_WITH_UNPARSEABLE_PARAMS = "200 Success with data:\r\n<something-unexpected />\r\n\r\n";
-var FAILURE_RESPONSE = "102 Failure";
-var FAILURE_RESPONSE_WITH_PARAMS = "102 Failure:\r\nsomething: 123\r\nsomething else: test\r\n\r\n";
-var ASYNC_RESPONSE = "512 Async event:\r\nprotocol version: 9.5\r\nmodel: xyz\r\ntime: 12:40:12\r\n\r\n";
-var INVALID_RESPONSE = "something invalid";
+// var SUCCESS_RESPONSE = "200 ok\r\n";
```

Comment:
can remove this and the other old ones in comments

Reply:
yeah , just kept for code review